### PR TITLE
Final merge of 5.2.5

### PR DESCRIPTION
**Pull request by tst-rwildes**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,6 +12,10 @@ The application suite consists of three core components:
 
 ### Releases
 
+[Release 5.2.5](https://github.com/google/earthenterprise/releases/tag/5.2.5-714.41)
+* [Release Notes](http://www.opengee.org/geedocs/5.2.5/answer/7160006.html)
```
Comment (tst-nfarah) [inline README.md]: we probably should start using https for opengee.org links

File: README.md [hunk 2]
```diff
@@ -20,18 +24,6 @@ The application suite consists of three core components:
...
-[Release 5.2.1](https://github.com/google/earthenterprise/releases/tag/5.2.1-6.final)
-* [Release Notes](http://www.opengee.org/geedocs/5.2.1/answer/7160001.html)
-* [Release Documentation](http://www.opengee.org/geedocs/5.2.1/)
-
-[Release 5.2.0](https://github.com/google/earthenterprise/releases/tag/5.2.0-3.final)
-* [Release Notes](http://www.opengee.org/geedocs/5.2.0/answer/7160000.html)
-* [Release Documentation](http://www.opengee.org/geedocs/5.2.0/)
-
 ### More information
 * [OpenGEE website](http://www.opengee.org)
```
Comment (tst-nfarah) [inline README.md]: same comment as above about https

File: docs/geedocs/5.2.5/answer/7160006.html [hunk 3]
```diff
@@ -38,23 +38,22 @@
...
-
   <h5>
     <p id="overview_5.2.5">New Features</p>
   </h5>
-    <p><strong>When a GEE server is queried for server definitions the response can be returned in proper JSON format.</strong> The <code>v</code> parameter can be added to query string to specify which version of the API will be returned. The valid options are <code>v=1</code> and <code>v=2</code>. Version one returns data in the current format. Version two returns data in JSON format. An example url which returns data in JSON format looks like: <code>http://&lt;GEE earth server&gt;/&lt;database_publish_point&gt;/query?request=Json&amp;v=2</code>.</p>
+    <p><strong>When a GEE server is queried for server definitions, the response can be returned in valid JSON format.</strong> The <code>v</code> parameter can be added to the query string to specify which version of the API to use. The options are <code>v=1</code> and <code>v=2</code>. Version <code>1</code> returns data in the current format. Version <code>2</code> returns valid JSON data. An example URL that returns data in JSON format is: <code>http://&lt;GEE earth server&gt;/&lt;database_publish_point&gt;/query?request=Json&amp;v=2</code>.</p>
```
Comment (tst-nfarah) [inline docs/geedocs/5.2.5/answer/7160006.html]: would it be possible to see how this html would looks like ? using github doc.

Comment (tst-rwildes) [reply docs/geedocs/5.2.5/answer/7160006.html]: image

Comment (tst-nfarah) [reply docs/geedocs/5.2.5/answer/7160006.html]: could you post the url to github docs ?

File: README.md (hunk 1 above)
Comment (tst-rwildes) [reply README.md]: changed

File: README.md (hunk 2 above)
Comment (tst-rwildes) [reply README.md]: changed

File: docs/geedocs/5.2.5/answer/7160006.html (hunk 3 above)
Comment (tst-rwildes) [reply docs/geedocs/5.2.5/answer/7160006.html]: https://www.opengee.org/geedocs/5.2.5/answer/7160006.html

**Pull request merged**